### PR TITLE
Add support for the Laravel asset() helpers

### PR DESCRIPTION
**Pull request by jaxwilko**
This PR adds some of the Laravel defaults into Winter's configs.

> `app.asset_url` was added with reference to [this PR](https://github.com/wintercms/storm/pull/67) which adds support for Laravel asset urls.

IMHO we should consider `env()` calls for every config that Laravel uses them for, which will be possible once https://github.com/wintercms/winter/pull/404 is merged. @bennothommo @LukeTowers thoughts?

<a href="https://gitpod.io/#https://github.com/wintercms/winter/pull/426"></a>

### CODE REVIEW

File: config/cache.php [hunk 1]
```diff
@@ -80,12 +80,12 @@
...
-            'secret'   => '',
-            'region'   => 'us-east-1',
-            'table'    => 'cache',
-            'endpoint' => '',
+            'driver'    => 'dynamodb',
+            'key'       => env('AWS_ACCESS_KEY_ID'),
+            'secret'    => env('AWS_SECRET_ACCESS_KEY'),
+            'region'    => env('AWS_DEFAULT_REGION', 'us-east-1'),
+            'table'     => env('DYNAMODB_CACHE_TABLE', 'cache'),
+            'endpoint'  => env('DYNAMODB_ENDPOINT'),
```
Comment (LukeTowers) [inline config/cache.php]:
```suggestion
            'driver'   => 'dynamodb',
            'key'      => env('AWS_ACCESS_KEY_ID'),
            'secret'   => env('AWS_SECRET_ACCESS_KEY'),
            'region'   => env('AWS_DEFAULT_REGION', 'us-east-1'),
            'table'    => env('DYNAMODB_CACHE_TABLE', 'cache'),
            'endpoint' => env('DYNAMODB_ENDPOINT'),
```

File: config/app.php [hunk 2]
```diff
@@ -43,6 +43,19 @@
...
 
+    /*
+    |--------------------------------------------------------------------------
+    | Asset URL
+    |--------------------------------------------------------------------------
+    |
+    | This URL is used to properly generate URLs for assets, including
+    | URLs generated by the `| theme` filter in twig. If set to null, the
+    | URL used will be the current hostname or `app.url` config.
+    |
```
Comment (LukeTowers) [inline config/app.php]:
```suggestion
    | This URL is used to properly generate URLs for assets, including
    | URLs generated by the `| theme` & `| asset` filters in Twig, or the
    | `Url::asset()` & `asset()` helpers. If set to null, the URL used will 
    | be the current hostname or `app.url` config.
    |
    |     'asset_url' => 'https://cdn.example.com/',
    |
```

File: config/app.php [hunk 3]
```diff
@@ -43,6 +43,22 @@
...
     'url' => 'http://localhost',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Asset URL
+    |--------------------------------------------------------------------------
+    |
+    | This URL is used to properly generate URLs for assets, including
+    | URLs generated by the `| theme` & `| asset` filters in Twig, or the
+    | `Url::asset()` & `asset()` helpers. If set to null, the URL used will
```
Comment (LukeTowers) [inline config/app.php]:
```suggestion
    | `Url::asset()` & `asset()` helpers. If set to null, the URL used will
```

**Comment by bennothommo** [comment]
> IMHO we should consider env() calls for every config that Laravel uses them for

I cannot "yes" this enough.

**Comment by LukeTowers** [comment]
> IMHO we should consider env() calls for every config that Laravel uses them for

I agree with that and once #404 is merged then the main blocker to that in the past will be able to be dealt with and we can have a configuration system that works well for newbies and pros. I'm excited for it :)

**Pull request merged**